### ISSUE TITLE
Intro to tensorflow.js

### ISSUE THREAD

**Issue by nileshpatra** [pydatadelhi/talks#98]
- Abstract (2-3 lines)
This will be a session on tensorflow.js , as it is getting pretty popular as a web framework for machine learning tools and thus serve models on the web .

- Brief Description and Contents to be covered
This will be an intro to tf.js and the various layered APIs that come along with it for leveraging the power of machine learning in the web . It will give an insight to how tf.js works and ends with an intro to ML5.JS which is another API for machine learning. Contents : 
:what is tensorflow.js
:how to use tensorflow.js
:Tensorflow graph visualisation
:ML5.JS : the next layer API.

- Pre-requisites for the talk
Familiarity with neural networks and JavaScript  
- Time required for the talk
20 to 30 minutes
- Link to slides
https://docs.google.com/presentation/d/1-1q1v87KYB4lfkLjT1mPjj2wotcKecXhYyJ2n32-1fg/edit?usp=sharing
- Will you be doing hands-on demo as well?
Yes 
- Link to ipython notebook (if any)
(Not ipython , but has the code)
https://github.com/nileshpatra/linear-regression-simulation-using-tensorflow.js/blob/master/lin_reg_simulation.js
- About yourself
I am a second year student at NSIT , New Delhi and I have been involved with machine learning for more than an year now . I have a lot of interest in deep learning and data analytics and I have made a few projects on the same . I'm also an avid open source contributor and regularly contribute code to open source orgs which also helps me grow as a developer .
- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Yes

**Comment by shagunsodhani** [comment]
Thank you @nileshpatra for proposing the talk. The topic is pretty interesting and I am looking forward to the slides :)

**Comment by nileshpatra** [comment]
Sure , I'll add them by tomorrow for sure .

**Comment by nileshpatra** [comment]
@shagunsodhani , added the link to the slides . Please do have a look :)

**Comment by shagunsodhani** [comment]
Hey @nileshpatra The slides are good! just 2 suggestions - some of the slides have typos (eg slide 8 "us" vs "use"). Since many people are familiar with tensorflow python api, it would be useful to have a slide comparing tf.js and tf python in terms of api or even just what functionalities are supported.

Best of luck for your talk.

**Comment by nileshpatra** [comment]
@shagunsodhani thanks for the suggestions , I'll definitely incorporate the changes . 
Also wanted to know , when is this talk scheduled?

**Comment by nileshpatra** [comment]
@shagunsodhani  , @MSanKeys963 , I have updated the slides according to the suggestions . Could you set this talk up in the next meet?

**Comment by sanketverma1704** [comment]
I'll let you know once we announce the meetup.

**Comment by nileshpatra** [comment]
@MSanKeys963 , as I can see that the accepted label is added , could you please confirm if I am in for the next talk? I know it's a bit early to ask , but I actually need to book a couple of tickets accordingly.

**Comment by sanketverma1704** [comment]
Thanks for the talk, closing this now.
